cmd/srcd/cmd/sql.go

```diff
@@ -192,81 +162,32 @@ func startGitbaseWithClient(client api.EngineClient) error {
 		return humanizef(err, "could not start gitbase")
 	}
 
-	if err := docker.EnsureInstalled(components.MysqlCli.Image, components.MysqlCli.Version); err != nil {
```

Comment:
this function has name `startGitbaseWithClient` but it doesn't start client anymore. Just gitbase.

Reply:
done [here](https://github.com/src-d/engine/pull/439/commits/50128fb356d9b925cd6a202479d9118bd765f772) and [here](https://github.com/src-d/engine/pull/439/commits/32b76ae48f5fe71af8df0d5c3bb286c7e2366f23).